<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>750</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>1000</height>
   </size>
  </property>
  <property name="font">
   <font>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Athenas Dash ECU</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(40, 42, 54);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>1199</width>
     <height>1000</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>950</width>
      <height>750</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>1200</width>
      <height>1000</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>DS-Digital</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(40, 42, 54);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_0_main">
     <widget class="QPushButton" name="btnPage2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>120</y>
        <width>250</width>
        <height>150</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DS-Digital</family>
        <pointsize>-1</pointsize>
        <bold>true</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">        QPushButton {
            background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0.05 #630303, stop: 1 #4cc7c9);
            border-radius: 8px;
            color: #ffffff;
            font-family: DS-Digital;
            font-size: 45px;
            font-weight: bold;
            padding: 13px 32px;
            text-decoration: none;
        }
        QPushButton:hover {
            background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0.05 #4cc7c9, stop: 1 #630303);
        }
        QPushButton:pressed {
            position:relative;
            top:1px;
        }</string>
      </property>
      <property name="text">
       <string>PAGE 2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPage3">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>120</y>
        <width>250</width>
        <height>150</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DS-Digital</family>
        <pointsize>-1</pointsize>
        <bold>true</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">        QPushButton {
            background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0.05 #630303, stop: 1 #4cc7c9);
            border-radius: 8px;
            color: #ffffff;
            font-family: DS-Digital;
            font-size: 45px;
            font-weight: bold;
            padding: 13px 32px;
            text-decoration: none;
        }
        QPushButton:hover {
            background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0.05 #4cc7c9, stop: 1 #630303);
        }
        QPushButton:pressed {
            position:relative;
            top:1px;
        }</string>
      </property>
      <property name="text">
       <string>PAGE 3</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPage1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>250</width>
        <height>150</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>-1</pointsize>
        <bold>true</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">        QPushButton {
            background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0.05 #630303, stop: 1 #4cc7c9);
            border-radius: 8px;
            color: #ffffff;
            font-family: Verdana;
            font-size: 25px;
            font-weight: bold;
            padding: 13px 32px;
            text-decoration: none;
        }
        QPushButton:hover {
            background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0.05 #4cc7c9, stop: 1 #630303);
        }
        QPushButton:pressed {
            position:relative;
            top:1px;
        }</string>
      </property>
      <property name="text">
       <string>PAGE 1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>561</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>ATHENAS DASH ECU - V0.1.12</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPage5">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>290</y>
        <width>250</width>
        <height>150</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DS-Digital</family>
        <pointsize>-1</pointsize>
        <bold>true</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">        QPushButton {
            background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0.05 #630303, stop: 1 #4cc7c9);
            border-radius: 8px;
            color: #ffffff;
            font-family: DS-Digital;
            font-size: 45px;
            font-weight: bold;
            padding: 13px 32px;
            text-decoration: none;
        }
        QPushButton:hover {
            background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0.05 #4cc7c9, stop: 1 #630303);
        }
        QPushButton:pressed {
            position:relative;
            top:1px;
        }</string>
      </property>
      <property name="text">
       <string>PAGE 5</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPage6">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>290</y>
        <width>250</width>
        <height>150</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DS-Digital</family>
        <pointsize>-1</pointsize>
        <bold>true</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">        QPushButton {
            background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0.05 #630303, stop: 1 #4cc7c9);
            border-radius: 8px;
            color: #ffffff;
            font-family: DS-Digital;
            font-size: 45px;
            font-weight: bold;
            padding: 13px 32px;
            text-decoration: none;
        }
        QPushButton:hover {
            background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0.05 #4cc7c9, stop: 1 #630303);
        }
        QPushButton:pressed {
            position:relative;
            top:1px;
        }</string>
      </property>
      <property name="text">
       <string>PAGE 6</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPage4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>290</y>
        <width>250</width>
        <height>150</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DS-Digital</family>
        <pointsize>-1</pointsize>
        <bold>true</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">        QPushButton {
            background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0.05 #630303, stop: 1 #4cc7c9);
            border-radius: 8px;
            color: #ffffff;
            font-family: DS-Digital;
            font-size: 45px;
            font-weight: bold;
            padding: 13px 32px;
            text-decoration: none;
        }
        QPushButton:hover {
            background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0.05 #4cc7c9, stop: 1 #630303);
        }
        QPushButton:pressed {
            position:relative;
            top:1px;
        }</string>
      </property>
      <property name="text">
       <string>PAGE 4</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_1">
     <widget class="QPushButton" name="bntReturn_1">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>310</y>
        <width>250</width>
        <height>150</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DS-Digital</family>
        <pointsize>-1</pointsize>
        <bold>true</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">        QPushButton {
            background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0.05 #630303, stop: 1 #4cc7c9);
            border-radius: 8px;
            color: #ffffff;
            font-family: DS-Digital;
            font-size: 45px;
            font-weight: bold;
            padding: 13px 32px;
            text-decoration: none;
        }
        QPushButton:hover {
            background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0.05 #4cc7c9, stop: 1 #630303);
        }
        QPushButton:pressed {
            position:relative;
            top:1px;
        }</string>
      </property>
      <property name="text">
       <string>RETURN</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>180</y>
        <width>351</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>PAGE 2</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="bntReturn_2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>260</y>
        <width>250</width>
        <height>150</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DS-Digital</family>
        <pointsize>-1</pointsize>
        <bold>true</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">        QPushButton {
            background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0.05 #630303, stop: 1 #4cc7c9);
            border-radius: 8px;
            color: #ffffff;
            font-family: DS-Digital;
            font-size: 45px;
            font-weight: bold;
            padding: 13px 32px;
            text-decoration: none;
        }
        QPushButton:hover {
            background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0.05 #4cc7c9, stop: 1 #630303);
        }
        QPushButton:pressed {
            position:relative;
            top:1px;
        }</string>
      </property>
      <property name="text">
       <string>RETURN</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>140</y>
        <width>351</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>PAGE 3</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QPushButton" name="bntReturn_3">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>20</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DS-Digital</family>
        <pointsize>-1</pointsize>
        <bold>true</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">        QPushButton {
            background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0.05 #630303, stop: 1 #4cc7c9);
            border-radius: 8px;
            color: #ffffff;
            font-family: DS-Digital;
            font-size: 45px;
            font-weight: bold;
            padding: 13px 32px;
            text-decoration: none;
        }
        QPushButton:hover {
            background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0.05 #4cc7c9, stop: 1 #630303);
        }
        QPushButton:pressed {
            position:relative;
            top:1px;
        }</string>
      </property>
      <property name="text">
       <string>Return</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>30</y>
        <width>281</width>
        <height>54</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DS-Digital</family>
        <pointsize>45</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>MEDIDORES</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="AnalogGaugeWidget" name="widgetSpeed" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>300</width>
        <height>300</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>600</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 0, 127);</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout"/>
     </widget>
     <widget class="AnalogGaugeWidget" name="widgetRPM" native="true">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>90</y>
        <width>300</width>
        <height>300</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>600</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 0, 127);</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2"/>
     </widget>
     <widget class="AnalogGaugeWidget" name="widgetTPS" native="true">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>90</y>
        <width>300</width>
        <height>300</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>600</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 0, 127);</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3"/>
     </widget>
     <widget class="AnalogGaugeWidget" name="widgetPW" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>410</y>
        <width>300</width>
        <height>300</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>600</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 0, 127);</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4"/>
     </widget>
     <widget class="AnalogGaugeWidget" name="widgetDuty" native="true">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>410</y>
        <width>300</width>
        <height>300</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>600</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 0, 127);</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5"/>
     </widget>
     <widget class="AnalogGaugeWidget" name="widgetMAP" native="true">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>410</y>
        <width>300</width>
        <height>300</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>600</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 0, 127);</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6"/>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AnalogGaugeWidget</class>
   <extends>QWidget</extends>
   <header location="global">analoggaugewidget</header>
   <container>1</container>
   <slots>
    <signal>valueChanged(int)</signal>
    <slot>update_angle_offset(int)</slot>
    <slot>update_value(int)</slot>
    <slot>update_value2(int)</slot>
    <slot>set_start_scale_angle(int)</slot>
    <slot>set_total_scale_angle_size(int)</slot>
    <slot>set_MinValue(int)</slot>
    <slot>set_MaxValue(int)</slot>
    <slot>set_NeedleColor(int)</slot>
    <slot>set_scala_main_count(int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
